Makefile

```diff
@@ -260,7 +260,27 @@ release-manager-snapshot: release
 .PHONY: release-manager-release
 release-manager-release: release
 
+JAVA_ATTACHER_VERSION:=1.26.0
```

Comment:
The next version (1.27.0) should be out very soon.

Reply:
How often does this version change? Would it be worth to think about an alternative of manually keeping this updated? (as a follow up ofc).

Reply:
Typically, we release a new minor version of the agent per minor version of the stack. However, we rarely do changes in the attacher itself.

Reply:
The release is out
```suggestion
JAVA_ATTACHER_VERSION:=1.27.0
```